Documentation/network-policies.md

```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
 In this example we are closing all the inbound communication on the namespace monitoring and just allowing the necessary traffic.
 **This example are only tested with calico provider.**
 
-Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https://kubernetes.io/docs/user-guide/networkpolicies) in you k8s cluster.
+Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) in you k8s cluster.
```

Comment:
First, follow the instructions to [add Calico to an existing Kubernetes cluster][add-calico]. Be certain to [enable network policy][enable-policy] in the cluster.

the bottom of the page gets these links:
[add-calico]: http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/
[enable-policy]: https://kubernetes.io/docs/user-guide/networkpolicies/

question: which policy?